<?xml version="1.0"?>
<launch>
  <!-- Parameters to pass to the ROS node -->
  <!-- <arg name="move_group" default="a_bot"/> -->
  
  <!-- This calls the ROS node we have written, using the parameters above -->
  <node name="precision_gripper_server" pkg="o2as_precision_gripper" type="precision_gripper_server.py" respawn="false" output="screen" />
  
  <node name="precision_gripper_client" pkg="o2as_precision_gripper" type="precision_gripper_client.py" respawn="false" output="screen">
    <!-- <param name="move_group" value="$(arg move_group)"/> -->
  </node>
  
</launch>
